addons/website_quote/controllers/main.py

```diff
@@ -117,8 +127,19 @@ def update(self, line_id, remove=False, unlink=False, order_id=None, token=None,
             return False
         number = -1 if remove else 1
         quantity = OrderLine.product_uom_qty + number
+        if quantity < 0:
+            quantity = 0
         OrderLine.write({'product_uom_qty': quantity})
-        return [str(quantity), str(Order.amount_total)]
+        currency = Order.currency_id
+
+        return {
+            'order_line_product_uom_qty': str(quantity),
```

Comment:
I'm pretty sure we only support integer quantities, currently they are displayed as float.